pkg/sfu/downtrack.go

```diff
@@ -376,9 +388,18 @@ func (d *DownTrack) Bind(t webrtc.TrackLocalContext) (webrtc.RTPCodecParameters,
 	d.bound.Store(true)
 	d.bindLock.Unlock()
 
-	d.forwarder.DetermineCodec(d.codec, d.receiver.HeaderExtensions())
+	// Bind is called under RTPSender.mu lock, call the RTPSender.GetParameters in goroutine to avoid deadlock
+	go func() {
+		if tr := d.transceiver.Load(); tr != nil {
+			extensions := tr.Sender().GetParameters().HeaderExtensions
+			d.params.Logger.Debugw("negotiated downtrack extensions", "extensions", extensions)
+			d.SetRTPHeaderExtensions(extensions)
```

Comment:
Is this going to cause a race?

Reply:
I think it would be ok that we only write the extension ids in there.